src/leo-client-app/src/components/navbar/Navbar.tsx

```diff
@@ -19,7 +19,7 @@ const navbarItems = [
     path: "/satellite-logs",
   },
   {
-    heading: "Log Out",
+    heading: "Account",
```

Comment:
keep this log out, we dont have account functionality so it'll be log out for now (until we get it working)